EIC-EA/EIC-community-D8
# EICNET-600: Create the group search block

### Description

* Add `search_group` page to `group_overviews` view
* Add permission `access group search` and update group and outsider roles.
* Add content fields to Search API: title (fulltext), field_body (fulltext), status
* Add sorts to Search API for search_group: search_api_relevance, content_created
* Add facet summary `group_search_summary` with facets for content_type, field_vocab_topics and field_vocab_geo
* Add `group_search` context

There are some issues, but not sure if this is related to my local installation where the Group PURL module is not fully functional. 

### Test

* The `access group search` permission should be removed from the `search_group` page to be able to view it.
* Add a group with some content (document, discussion & gallery)
* Go to `/search-group/{group_id}` to view the search page (selecting filters will remove the {group_id} from the URL, you can manually add it again and the filters will do their job)
